src/libostree/ostree-repo-private.h

```diff
@@ -415,6 +419,28 @@ _ostree_repo_get_remote_inherited (OstreeRepo  *self,
 
 #ifndef OSTREE_ENABLE_EXPERIMENTAL_API
 
+typedef enum {
+  OSTREE_REPO_LOCK_SHARED,
+  OSTREE_REPO_LOCK_EXCLUSIVE
+} OstreeRepoLockType;
```

Comment:
Worthwhile to add a comment mentioning that this is duplicated in `ostree-repo.h`, should be kept in sync, and should be removed when the API graduates from being experimental.

Reply:
Good point. I'll add that comment.